File: scratch/find_page_views.py
import re

with open(r'c:\Users\diwas\OneDrive\Documents\Desktop\school management saraswati\html\admin-portal.html', 'r', encoding='utf-8', errors='ignore') as f:
    content = f.read()

# Find all divs with class="page-view" and print their ID and start line
matches = re.finditer(r'<div\s+[^>]*class=["\']page-view[^"\']*["\'][^>]*id=["\']([^"\']+)["\']', content)
for m in matches:
    print(f"Page ID: {m.group(1)}")

File: scratch/find_page_views.js
const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\diwas\\OneDrive\\Documents\\Desktop\\school management saraswati\\html\\admin-portal.html', 'utf8');

const regex = /<div\s+[^>]*class=["']page-view[^"']*["'][^>]*id=["']([^"']+)["']/g;
let match;
while ((match = regex.exec(content)) !== null) {
  console.log(`Page ID: ${match[1]}`);
}
